packages/image-generation/tests/test_image_generator_md_structure.py

```
@@ -224,6 +250,53 @@ def test_forbids_coding_model_example(self) -> None:
+    def test_full_prompt_allows_english_technical_terms_and_proper_nouns(self) -> None:
+        full_prompt = _extract_full_prompt_body(STEP2).lower().replace("\\\n", "")
+        assert "technical terms" in full_prompt
+        assert "proper nouns" in full_prompt
+        assert "may remain in english" in full_prompt
+
+    def test_explicit_user_language_takes_precedence(self) -> None:
+        full_prompt = _extract_full_prompt_body(STEP2).lower()
+        assert "user's prompt explicitly" in full_prompt
+        assert "follow that request instead" in full_prompt
+        # 逆検証: FULL_PROMPT 代入式の"後"にある Step 2 の説明文が
+        # 抽出結果に混入していないこと（抽出が貪欲すぎないことのガード）。
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)